doc/source/api/core/solver/settings.rst

```diff
@@ -1,77 +1,76 @@
 .. _ref_settings:
 
-Settings objects
-================
-Settings objects provide a natural way to access and modify Fluent settings and
-issue commands with a hierarchy of objects.
+Solver settings objects
+=======================
+Solver settings objects provide a natural way to access and modify Fluent solver
+settings and issue commands to be executed in the Fluent solver.
 
-Top-level settings objects
---------------------------
-The top-level settings object is available as the :ref:`root<settings_root_section>`
-property of ``session.solver``.
+Accessing solver settings
+-------------------------
+An appropriate call to ``launch_fluent`` returns an object (named ``solver`` in
+the following code snippets) whose interface directly exposes the
+:ref:`root<settings_root_section>` of the solver settings hierarchy.
 
 .. code-block::
 
   >>> import ansys.fluent.core as pyfluent
   >>> solver = pyfluent.launch_fluent(mode="solver")
-  >>> root = solver
 
 
-The ``root`` object contains attributes such as ``file``, ``setup``,
-``solution``, and ``results``.  These objects are also instances of settings
-objects and roughly mirror the outline view in Fluent.
+The ``solver`` object contains attributes such as ``file``, ``setup``, ``solution`` and
+``results``, which are also instances of settings objects. Note that the last three are
+top-level nodes in the outline tree view in Fluent's graphical user interface (GUI) - much
+of this settings hierarchy has been designed in close alignment with that GUI hierarchy.
```

Comment:
```suggestion
of this settings hierarchy has been designed in close alignment with this GUI hierarchy.
```

Reply:
It looks like I didn't comment correctly here. The intent is to avoid a long sentence. I wanted to recommend breaking it out into two sentences rather than using dashes.